### PR TITLE
Document band specification edition

### PR DESCRIPTION
**Pull request by restaste**
This expands the raster management example by adding an example of image bandspec edition.

### CODE REVIEW

File: examples/raster_management.py [hunk 1]
```diff
@@ -21,8 +21,18 @@
...
+# Upload, edition and removal
 local_raster_id = client.upload_raster("data/raster1.tif", name="A short-lived raster")
 print("Uploaded a second local raster=", local_raster_id)
+# Editing the image's band specification. See https://docs.picterra.ch/imagery/#Multispectral
+client.edit_raster(local_raster_id, multispectral_band_specification={
+    "ranges": [
+        [0, 128], [0, 128], [0, 128]
+    ],
+    "display_bands": [
+        {"name": "default", "bands": [2, 1, 0]}
```
Comment (ao-picterra) [inline examples/raster_management.py:33]: nit: would be worth to add the `type` key?

File: examples/raster_management.py [hunk 2]
```diff
@@ -21,8 +21,18 @@
...
+    ]
+})
+# Deleting the image
 client.delete_raster(local_raster_id)
-print("Deleted raster=", local_raster_id)
+print("Deleted raster=", local_raster_id)
```
Comment (ao-picterra) [inline examples/raster_management.py:38]: nit: missing newline diff?

**Review by ao-picterra — CHANGES_REQUESTED** [review]

File: examples/raster_management.py (hunk 2 above)
Comment (stephanerestani-picterra) [reply examples/raster_management.py]: Ah yep. Added that back.

File: examples/raster_management.py (hunk 1 above)
Comment (stephanerestani-picterra) [reply examples/raster_management.py]: Good catch! Added

**Review by ao-picterra — APPROVED** [review]

**Pull request merged**